### PR TITLE
nix-build: add page

### PR DESCRIPTION
**Pull request by azuzunaga**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/nix-env.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Upgrade all packages:
 
 `nix-env -u`
+
+- Install a package from a URL:
+
+`nix-env -i {{pkg_name}} --file {{URL}}`
```
Comment (sbrl) [inline pages/common/nix-env.md]: Perhaps we could have a concrete example of a URL?

If the domain varies, perhaps `example.com` would be a good idea.

Comment (azuzunaga) [reply pages/common/nix-env.md]: The domain name does vary. I'll use `example.com`.

Comment (azuzunaga) [reply pages/common/nix-env.md]: @sbrl, fixed.

File: pages/common/nix-env.md [hunk 2]
```diff
@@ -30,3 +30,7 @@
 - Upgrade all packages:
 
 `nix-env -u`
+
+- Install a package from a URL:
```
Comment (agnivade) [inline pages/common/nix-env.md]: Let's move this to after the "Install package" example

Comment (sbrl) [reply pages/common/nix-env.md]: Hrm, looks like Travis CI is having a hiccup moment @azuzunaga. We can force-mergethis is necessary.

Comment (azuzunaga) [reply pages/common/nix-env.md]: @sbrl, I rebased off the latest changes on master and force pushed to restart the travis ci build, seems to have worked 🙂

**Comment by sbrl** [comment]
Merging, since this has been around a while and @agnivade's comment appears to have been addressed.

**Pull request merged**